src/IPinfo/Utilities/CountryHelper.cs

```diff
@@ -30,10 +31,32 @@ internal static class CountryHelper
             return countries;
         });
 
+        /// <summary>
+        /// Lazy initialization for the EUcountry List from EUcountry json file.
+        /// </summary>
+        private static readonly Lazy<List<string>> eu_countries =
```

Comment:
Just to be on consistent convention for [private static](https://docs.microsoft.com/en-us/dotnet/csharp/fundamentals/coding-style/coding-conventions#camel-case), `eu_countries` to be replaced with `s_euCountries`.